### PR TITLE
lightning: save NEXT_GLOBAL_ROW_ID as uint64 instead of int64

### LINKED ISSUE #45427: lightning wrongly use int64 to save NEXT_GLOBAL_ROW_ID
## Bug Report

Please answer these questions before submitting your issue. Thanks!

### 1. Minimal reproduce step (Required)

https://github.com/pingcap/tiflow/issues/9391#issuecomment-1639704693

https://github.com/pingcap/tidb/blob/f1b9da103166162944ae41dc0943c93c0b91923b/br/pkg/lightning/backend/tidb/tidb.go#L851

should use unit64 here

### PR DESCRIPTION
**Pull request by lyzx2001**
### What problem does this PR solve?

Issue Number: close #45427 

Problem Summary:

lightning wrongly use `int64` to save `NEXT_GLOBAL_ROW_ID`.

https://github.com/pingcap/tiflow/issues/9391#issuecomment-1639704693

### What is changed and how it works?

https://github.com/pingcap/tidb/blob/f1b9da103166162944ae41dc0943c93c0b91923b/br/pkg/lightning/backend/tidb/tidb.go#L847C1-L852

`nextID`'s type is changed to `string`.

### Check List

Tests 

- [ ] Unit test
- [x] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

```release-note
None
```

### CODE REVIEW

File: br/pkg/lightning/backend/tidb/tidb.go [hunk 1]
```diff
@@ -870,7 +870,7 @@ func (*Writer) IsSynced() bool {
 // TableAutoIDInfo is the auto id information of a table.
 type TableAutoIDInfo struct {
 	Column string
-	NextID int64
+	NextID string
```
Comment (lance6716) [inline br/pkg/lightning/backend/tidb/tidb.go:873]: it should be uint64. But seems no one is using this member? We can directly delete it.

Comment (lyzx2001) [reply br/pkg/lightning/backend/tidb/tidb.go:873]: https://github.com/pingcap/tidb/blob/ac0c90e232436573e1765ba944011eea028b9996/br/pkg/lightning/backend/tidb/tidb.go#L911-L915

will use this `NextID`. So if we change it to uint64, this `append` would result in error "Cannot use 'nextID' (type string) as the type uint64"

File: br/pkg/lightning/backend/tidb/tidb.go [hunk 2]
```diff
@@ -883,8 +883,7 @@ func FetchTableAutoIDInfos(ctx context.Context, exec utils.QueryExecutor, tableN
 	var autoIDInfos []*TableAutoIDInfo
 	for rows.Next() {
 		var (
-			dbName, tblName, columnName, idType string
-			nextID                              int64
```
Comment (lance6716) [inline br/pkg/lightning/backend/tidb/tidb.go:887]: can change the type of `nextID` to uint64, so SQL drvier will do the conversion work.

Comment (lyzx2001) [reply br/pkg/lightning/backend/tidb/tidb.go:887]: modified

**Review by lance6716 — APPROVED** [review]

**Review by D3Hunter — APPROVED** [review]

**Pull request merged**